aries-site/src/pages/components/card/navigational-card.mdx

```diff
@@ -15,7 +15,7 @@ Navigational cards enable the user to identify, compare, and choose their desire
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/ActivitiesNavigationalCards.js"
   docs="https://v2.grommet.io/card?theme=hpe#props"
   figma="https://www.figma.com/file/je5AYzMavRt1cG7rmw8j1W/HPE-Card-Component?node-id=1%3A2"
-  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Card/Card.js"
+  grommetSource="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/components/card/ActivitiesNavigationalCards.js"
```

Comment:
@luketa8 Can you leave the `grommetSource` links as they were actually? We use this to link back to the Grommet component source code

Reply:
Done